tutorials/performance/cpu_optimization.rst

```diff
@@ -225,8 +225,8 @@ SceneTree
 
 Although Nodes are an incredibly powerful and versatile concept, be aware that
 every node has a cost. Built-in functions such as `_process()` and
-`_physics_process()` propagate through the tree. This housekeeping can reduce
-performance when you have very large numbers of nodes (usually in the thousands).
+`_physics_process()` propagate through the tree. This housekeeping does start to reduce
+performance when you have very large numbers of nodes (depending on platform, but about 10% of frame time per ten thousand nodes on a desktop CPU).
```

Comment:
I'm not sure if giving an "accurate" figure is a good idea here. People may start extrapolating based on this in weird ways, especially since the CPU model isn't mentioned.

Reply:
Mmmm, agreed. I however have lived for two years worrying about having 1000 nodes in my tree, which as it seems is completely unnecessary. So it may also lead to people trying to over optimize causing slowdown in production time. I do get your point and agree with you when I think about it, perhaps "does start to" is better than "can" but other than that we can close this PR